src/components/shared/__tests__/Validations.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import 'jest-enzyme';
+
+import { RegularNode } from '@stoplight/json-schema-tree';
+import { mount } from 'enzyme';
+import * as React from 'react';
+
+import { Validations } from '../../shared';
+import { getValidationsFromSchema } from '../Validations';
+
+describe('Validations component', () => {
+  it('should render number type validations', () => {
+    const node = new RegularNode({
+      type: 'number',
+      minimum: 10,
+      maximum: 40,
+      multipleOf: 10,
+      default: 20,
+      example: 20,
+      enum: [10, 20, 30, 40],
+    });
+
+    const validations = getValidationsFromSchema(node);
+    const wrapper = mount(<Validations validations={validations} />);
+
+    expect(wrapper).toIncludeText('>= 10<= 40');
+    expect(wrapper).toIncludeText('Allowed values:10203040');
+    expect(wrapper).toIncludeText('Default value:20');
+    expect(wrapper).toIncludeText('Multiple of value:10');
+    expect(wrapper).toIncludeText('Example value:20');
```

Comment:
I see you're attempting to use the good approach even with the not-so-good framework. We can likely discuss introduction of RTL in here later. Looks good for now.